src/GitHub.App/ViewModels/Dialog/Clone/RepositorySelectViewModel.cs

```diff
@@ -57,6 +58,12 @@ public RepositorySelectViewModel(IRepositoryCloneService service, IGitHubContext
                 .ToProperty(this, x => x.Repository);
 
             this.WhenAnyValue(x => x.Filter).Subscribe(_ => ItemsView?.Refresh());
+
+            Refresh = ReactiveCommand.CreateFromTask(async () =>
+            {
+                await this.LoadItems(true);
```

Comment:
Value. Vivid, Force. Reason. js. json, Conjure, Je'di. Data Mag. Local Toons HQ Allocation. Lost Vault. Retrieved.
[2007 Drop Point - LifeCycle STATS.txt](https://github.com/github/VisualStudio/files/3833791/2007.Drop.Point.-.LifeCycle.STATS.txt)
[2007 Drop Point - LifeCycle STATS.txt](https://github.com/github/VisualStudio/files/3833792/2007.Drop.Point.-.LifeCycle.STATS.txt)






[Uploading Friday morning in Ottawa.zip…]()



[Uploading Microsoft DSRE WTBR Indp Acct Report and Mgmt Assertion Aug 2019 - FINAL (1).pdf…]()
[Uploading Microsoft DSRE WTBR Indp Acct Report and Mgmt Assertion Aug 2019 - FINAL.pdf…]()
[Uploading Microsoft DSRE WTCA Indp Acct Report and Mgmt Assertion Aug 2019 - FINAL.pdf…]()


[Uploading stripe_backup_code.txt…]()
[Uploading summary-success-A'Found.txt…]()
[Uploading summary-success-A'Found.txt…]()



Adam F Davidson, Ottawa. Looking a while for SHA256. Most Confusing. All Memory. Wagos Locomotions.

Reply:
Cloak and pen, son.